README.md

```diff
@@ -1,110 +1,136 @@
+
+* **Vendor-neutral**: Fully compatible with the Prometheus, OpenTelemetry, and
+  Grafana open source ecosystems.
+* **Every signal**: Collect telemetry data for metrics, logs, traces, and
+  continuous profiles.
+* **Scalable**: Deploy on any number of machines to collect millions of active
+  series and terabytes of logs.
+* **Battle-tested**: Grafana Agent extends the existing battle-tested code from
+  the Prometheus and OpenTelemetry Collector projects.
+* **Powerful**: Write programmable pipelines with ease, and debug them using a
+  [built-in UI][UI].
+* **Batteries included**: Integrate with systems like MySQL, Kubernetes, and
+  Apache to get telemetry that's immediately useful.
+
+[UI]: https://grafana.com/docs/agent/latest/flow/monitoring/debugging/#grafana-agent-flow-ui
 
 ## Getting started
 
-When using Kubernetes this [link](https://grafana.com/docs/grafana-cloud/quickstart/agent-k8s) offers the best guide.
+Check out our [documentation][] to see:
 
-Other installation methods can be found in our
-[Grafana Agent](https://grafana.com/docs/agent/latest/set-up/) documentation.
+* [Installation instructions][] for Grafana Agent
+* Details about [Grafana Agent Flow][]
+* Steps for [Getting started][] with Grafana Agent Flow
+* The list of Grafana Agent Flow [Components][]
 
-More detailed [documentation](./docs/README.md) is provided as part of the
-repository.
+[documentation]: https://grafana.com/docs/agent/latest/
+[Installation instructions]: https://grafana.com/docs/agent/latest/set-up/
+[Grafana Agent Flow]: https://grafana.com/docs/agent/latest/flow/
+[Getting started]: https://grafana.com/docs/agent/latest/flow/getting_started/
+[Components]: https://grafana.com/docs/agent/latest/flow/reference/components/
 
 ## Example
 
-The [`example/`](./example) folder contains docker-compose configs and a local
-k3d/Tanka environment. Both examples deploy the Agent, Cortex and Grafana for
-testing the agent. See the [docker-compose README](./example/docker-compose/README.md)
-and the [k3d example README](./example/k3d/README.md) for more information.
+```river
+// Discover Kubernetes pods to collect metrics from.
+discovery.kubernetes "pods" {
+  role = "pod"
+}
+
+// Collect metrics from Kubernetes pods.
+prometheus.scrape "default" {
+  targets    = discovery.kubernetes.pods.targets
+  forward_to = [prometheus.remote_write.default.receiver]
+}
+
+// Get an API key from disk.
+local.file "apikey" {
+  filename  = "/var/data/my-api-key.txt"
+  is_secret = true
+}
+
+// Send metrics to a Prometheus remote_write endpoint.
+prometheus.remote_write "default" {
+  endpoint {
+    url = "http://localhost:9009/api/prom/push"
+
+    basic_auth {
+      username = "MY_USERNAME"
+      password = local.file.apikey.content
+    }
+  }
+}
+```
+
+We maintain an example [Docker Compose environment][] that can be used to
+launch dependencies to play with Grafana Agent locally.
+
+[Docker Compose environment]: ./example/docker-compose/
```

Comment:
Same as the installation instructions, let's start thinking about making this Flow-first, too?